draft-scenarios/coops/close_to_enemies.tex

```diff
@@ -0,0 +1,220 @@
+% !TeX spellcheck = en_US
+\addscenariosection{1}{Cooperative Scenario}{Close to enemies}{\images/balistic.png}
+
+\begin{multicols}{2}
+
+\textbf{Author:} Kelghor
+
+\textbf{Version:} 1.5
+
+\textbf{Source:} \href{https://discord.com/channels/740870068178649108/1313651865648369725}{Archon Studio Discord}
+
+\textit{"Sir! They're here again!"}
+
+\textit{"Sound the alarm, we'll repel them as always!" you command, and go to defend the city. Just as you have done many times before. Such is the hard life of the surrounding clans, with constant wars and skirmishes with nearby neighbors. Fortunately, your alliance is strong, so you know your back is covered.}
+
+\textit{Therefore, the time has come to end the long-standing wars and crush your enemies like a bug once and for all!}
+
+\subsection*{\MakeUppercase{Scenario Length}}
+
+This Scenario is played over 7 Rounds.
+
+\subsection*{\MakeUppercase{Player Setup}}
+
+\textbf{Player Count:} 1-6
+
+\textbf{Starting Resources:} 14 \svg{gold}, 5 \svg{building_materials}, 2 \svg{valuables}
+
+\textbf{Starting Income:} 10 \svg{gold}, 2 \svg{building_materials}, 0 \svg{valuables}
+
+\textbf{Starting Units:}
+\begin{itemize}
+  \item  1 × A Few \svgunit{bronze} Units of your choice
+\end{itemize}
+
+\textbf{Town Buildings:} \svgunit{bronze} Dwelling.
+
+\textbf{Additional Bonus:}: Only base bonus by difficulty (see the main rules).
+
+\textbf{Map Tile Pool:} Take $P$ × random Far (II–III) map Tiles as shared pool. None of them must contain a settlement. ($P$ stands for the number of players)
+
+\subsection*{\MakeUppercase{Map Setup}}
+
+Take the following Map Tiles and arrange them as shown in the Scenario map layout.
+Setup Far (II--III) Map Tiles with settlements of same Faction as players Towns as show a color Cube, or see example.
+
+\begin{itemize}
+  \item $P$ × Starting (I) Map Tile.
+  \item $P$ × Far (II--III) Map Tile with Settlement of same Faction as players Towns.
+  \item 2$P$ × Far (II--III) Map Tile - must not contain Tiles with Settlements.
+\end{itemize}
+
+\textit{Optional: You can distribute random pieces and covered pieces on the map so that they match your Faction's basic terrain.}
```

Comment:
That players can play this optional rule, and covered tiles place that same faction terrain will close to faction town tile... 

And random tiles will be known which tile is which terrain  so when player explore tile from pool he will explore his terrain. Im not sure that your wording is better. 

word "distribute" fit good i think. 

But maybe there could be longer & clearer way how to say it.

In fact it means, when players want, they distribute map tiles in way, that each will have  same mines in his hero trace, cuz each terrain has all types of mines...

Reply:
All right, well, it's still not entirely clear to me. Let me think and come up with something else.

Reply:
"Players can take map tiles of their Faction and place them on the map according to the template so that they are connected together. They keep the last Faction tile with them and can use it during exploration. This option leads to an even distribution of mines."

Reply:
Oh, if your intention is to have a just and even distribution of mines, then this should be clearly stated at the beginning.
> Optional: For an even distribution of mines...

Reply:
How about something like this?

_To ensure a balanced distribution of Mines, players may place Map Tiles matching their Faction onto the Map Template, ensuring all placed Tiles are connected. Each player keeps their final Faction Tile for use during exploration._

But honestly, I'm still not getting the idea of this rule. Examples shown in pictures follow this rule or not? Or it's impossible to tell?

Reply:
> But honestly, I'm still not getting the idea of this rule

I assume there are two advantages of this optional rule if it's used:
1. Gameplay/balance — each player is guaranteed to have one gold mine, one crystal mine and one stone mine.
2. Aesthetic/thematic — each player moves through their faction's own terrain, and also is more likely to utilize faction-specific mechanics using related map locations.

Reply:
> They [players] keep the last Faction tile with them and can use it during exploration.

In that case it's not a *shared* pool of tiles anymore.

Reply:
OK, let me get this straight now. There is an option to make a shared pool of tiles without settlements. I take it that while discovering new tiles not yet on the map, you draw from the pool at random. This optional rule basically cancels the shared pool, and every player gets one tile native to the home terrain of their faction?

Reply:
> This optional rule basically cancels the shared pool, and every player gets one tile native to the home terrain of their faction?

It affects non-pool tiles too. And yes, among other things, it also kinda makes the shared pool not random and actually more like 1 tile per player. But maybe there is still an option to explore other player's tiles theoretically. Might need @Kelghor to explain their vision further.

Reply:
Hi, Dima793 understund the purpose 100% it can be eastetic and balanced distribution of Mine.

If is used this option, random pool is out.

"OK, let me get this straight now. There is an option to make a shared pool of tiles without settlements. I take it that while discovering new tiles not yet on the map, you draw from the pool at random. This optional rule basically cancels the shared pool, and every player gets one tile native to the home terrain of their faction?"
Yes its true

Reply:
In that case, I think it makes more sense to keep it in a single place, rather than introducing Map Tile Pool, only to cancel it somewhere else. How about describing it in the Map Tile Pool section only, stating that each player gets a Far Map Tile matching the home terrain of their Faction, and mentioning that OPTIONALLY players can form a shared pool, to spice things up?

Reply:
Of course, I'd love to see the purpose, too. You could mention that the former guarantees balanced distribution of mines, while the latter adds more randomness.

Reply:
So reworked it, this is outdated, pls check new version and create new discussion?